test/controllers/locks_controller_test.rb

```diff
@@ -118,6 +119,33 @@ def create_lock(resource = nil, options = {})
       end
     end
 
+    describe '#create with PRODUCTION_STAGE_LOCK_REQUIRES_ADMIN' do
```

Comment:
if it's not too complicated I'd prefer this under `#create` as `describe "with PRODUCTION_STAGE_LOCK_REQUIRES_ADMIN"`